packages/tuffex/packages/components/src/switch/style/index.scss

```
@@ -1,10 +1,26 @@
+  inherits: false;
+  initial-value: 0deg;
+}
+
 .tuff-switch {
   // Override points. The track visuals moved off the root when the label was
   // added, so `.tuff-switch.is-active { background-color }` no longer reaches
   // them — these variables are the supported way to recolor a switch.
   --tuff-switch-track-color: var(--tx-fill-color, #f0f2f5);
   --tuff-switch-active-color: var(--tx-color-primary, #409eff);
   --tuff-switch-thumb-color: var(--tx-text-color-secondary, #909399);
+  // Both loading cues read against whatever the track currently is. The grey
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)